lib/runner.js

```diff
@@ -73,10 +77,11 @@ var setupTests = (log, args) => () => {
                 "modifyingTests",
                 "Generating elm.json for coverage at " + tmpElmJson + "..."
             );
-            var covSrc = path.resolve(path.join(coverageDir, args.path));
-            var originalPath = path.resolve(args.path);
 
             elmPackage["name"] = "author/project";
+            elmPackage["source-directories"] = elmPackage["source-directories"]
+                .map(sourceDir => fixOutOfRootPath(sourceDir));
+            elmPackage["source-directories"].push("elm-coverage-src");
```

Comment:
This is a bit less likely to collide. (Instead of `src/Coverage.elm` the user would have to have a source directory `elm-coverage-src` and inside `Coverage.elm`.)